.phpmetrics.yml

```
@@ -0,0 +1,94 @@
+        enabled: false  # Enable if you want historical metrics
+        
+    # Group metrics by namespace
+    groups:
+        - name: "Authentication"
+          match: "OpenFGA\\Authentication\\*"
+        - name: "Models"
+          match: "OpenFGA\\Models\\*"
+        - name: "Network"
+          match: "OpenFGA\\Network\\*"
+        - name: "Requests/Responses"
+          match: "OpenFGA\\(Requests|Responses)\\*"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)